packages/sak-ukjent-adresse/src/UkjentAdresseMeldingIndex.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FunctionComponent } from 'react';
+import { FormattedMessage, RawIntlProvider } from 'react-intl';
+
+import { createIntl } from '@navikt/ft-utils';
+import { AksjonspunktHelpTextHTML } from '@navikt/ft-ui-komponenter';
+
+import messages from '../i18n/nb_NO.json';
+import styles from './UkjentAdresseMeldingIndex.less';
+
+const intl = createIntl(messages);
+
+/**
+ * UkjentAdresseMeldingIndex
+ *
+ * Har ansvar for å vise risikoklassifisering for valgt behandling
+ * Viser en komponent som informerer saksbehandlere at brukeren i behandlingen
+ * mangler registrert adresse.
+ */
+const UkjentAdresseMeldingIndex: FunctionComponent = () => (
+  <RawIntlProvider value={intl}>
+    <div className={styles.ukjentAdresseMargins}>
+      <AksjonspunktHelpTextHTML>
+        <b><FormattedMessage id="UkjentAdresse.Text" /></b>
```

Comment:
I staden for <b></b> så brukar me typografi-komponentar fra nav-frontend. Sjå https://old-design-nav.vercel.app/designsystem/components/typografi/technical . Her er kanskje Element den du ser etter

Reply:
Ja det er det me brukar. Men dei driv å lagar noko nytt, men det ser ut som det framleis er i beta: https://aksel.nav.no/designsystem. Så me får ein jobb med å gå over til det nye etterkvart